Orange/widgets/visualize/owlineplot.py

```diff
@@ -543,7 +545,10 @@ def set_data(self, data):
             self.group_view.setEnabled(len(self.group_vars) > 1)
             self.group_var = self.data.domain.class_var \
                 if self.data.domain.has_discrete_class else None
-
+            self.info.set_input_summary(len(self.data),
+                                        format_summary_details(self.data))
```

Comment:
I'd rather show the info before the data is changed `check_data()`.
If one, for instance, inputs the `zoo` dataset, no information is shown.